doc/api/cluster.markdown

```diff
@@ -302,30 +643,30 @@ because of exiting or being signaled). Otherwise, it returns `false`.
 * `signal` {String} Name of the kill signal to send to the worker
   process.
 
-This function will kill the worker. In the master, it does this by disconnecting
-the `worker.process`, and once disconnected, killing with `signal`. In the
-worker, it does it by disconnecting the channel, and then exiting with code `0`.
+This function will, by default, signal the worker process to terminate. In the
+master process, this is done by first disconnecting the worker process, then
+sending the `signal` (which defaults to `SIGTERM`). Within the worker, the IPC
+channel is disconnected and the process exits with code `0`.
 
-Causes `.suicide` to be set.
+When called, the `worker.suicide` property will be set to `true`.
 
 This method is aliased as `worker.destroy()` for backwards compatibility.
```

Comment:
It should never have been changed, it was an error, AFAICT from examining the logs. destroy is what it actually does, it is not sufficiently like child_process.kill() or process.kill() to be called kill - though this is easy to miss from the function signature. We should switch the docs back to destroy, and mention its called kill for backwards compat reasons. I keep intending to PR this change, but I will if you think it our of scope here, as it probably is.